provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -287,15 +292,25 @@ def __init__(
 
         if args.url:
             # Lambda URL
-            self.function_url = aws.lambda_.FunctionUrl(
+            self.function_url_resource = aws.lambda_.FunctionUrl(
```

Comment:
Maybe we should keep binding to self.function_url but instead rename the output to `url` ?

This way if anybody needs to access cors or authorization_type on the `function_url` they still can (function_url_resource is kinda long), and we have a nice and simple output `my_func.url`

Reply:
Maybe `url_enabled` ?